Libs/MRML/Widgets/qMRMLLayoutManager_p.h

```diff
@@ -155,6 +155,7 @@ public slots:
   /// Top-level windows created and managed by this class that host additional viewports
   /// (outside the main application window).
   QMap<QString, ViewportInfo>  DetachedViewports;
+  int GlobalPauseRenderCount{ 0 };
```

Comment:
It is not quite global, just all the views that this layout manager owns. Could you rename it to "AllViewsPauseRenderCount" or something similar?